gridlab-d/gridlab-d
# Bugfix/1249

#### What's this Pull Request do?
Fixes HVAC breaker issues, where it will not allow the HVAC to continue to run after the breaker trips.

#### Where should the reviewer start?
Examine the source code and new autotest.

#### How should this be tested?
Run the new test_house_HVAC_breaker_trip.glm as an example to see the temperature continue to rise after the breaker trips.  If "hvac_breaker_rating" to something higher (like 400 A), the HVAC should operate normally.

#### Any background context you want to provide?
The related issue (#1213) still needs to be addressed.  It's effectively the same issue, just for explicit enduses instead of the HVAC.

#### Questions:
Commit logs.  Not much more needed.
